.gitignore

```diff
@@ -0,0 +1,4 @@
+Data/
```

Comment:
Maybe it'd be good to add a Data folder to the version control at some point. Not all data though (a few Mo max).

Reply:
Please also add .pyc files to the gitignore